docs/developers/contributing.md

```diff
@@ -210,6 +217,11 @@ You can then make a
 
 ## Building the docs
 
+Install the documents tools:
+```sh
+pip install -e ".[docs]"  # (quotes only needed for zsh shell)
+```
+
```

Comment:
This information already is in the documentation

 https://github.com/napari/napari/blob/4aef733d133f597c8ba12c7e47e9039f10d8fb17/docs/developers/documentation/index.md?plain=1#L10-L13

I think that duplication is not a good idea and if it is not clear then this section from documentation contribution docs should be updated.

Reply:
I agree that duplication is not good, but don't think that the description of building docs in the Contributing guide stands alone without inclusion of this instruction.  

It seems like by nature, the prerequisites section of Contributing Documentation is a condensed summary of the Contributing Guide. Maybe the solution to avoid duplication is to remove any specific details from the Contributing Documentation page and refer the reader to the Contributing guide.

Reply:
I don't have a strong opinion here but I agree that the two pages should somehow be harmonized — either move ~all the information to the "contributing documentation" page, and keep only a link to it from the contributing guide, or move it all to the contributing guide. I kinda favour the former, since they are quite different beasts.